pyfr/integrators/dual/pseudo/pseudocontrollers.py

```diff
@@ -140,8 +146,30 @@ def __init__(self, *args, **kwargs):
                     )
                 )
 
+        self.bind_dtau()
+
         self.backend.commit()
 
+    def adjust_dtau(self, dt):
+        old_dtau = self.dtau
+        self.dtau = dt / self._dt_dtau_ratio
+        self.multiply_limits(self.dtau / old_dtau)
+
+    def multiply_limits(self, y):
```

Comment:
Why is this its own function?

Reply:
As a result of what @WillTrojak said in 
https://github.com/PyFR/PyFR/pull/335#discussion_r1321524509

Reply:
At the moment the function is only called once and so adds little to no value.